src/keycloak/chart/values.yaml

```diff
@@ -15,12 +15,20 @@ domain: "###ZARF_VAR_DOMAIN###"
 # The primary Keycloak realm
 realm: uds
 
-# UDS Identity Config Environment Variables. More info here: https://github.com/defenseunicorns/uds-identity-config/blob/main/docs/CUSTOMIZE.md#templated-realm-values
+# UDS Identity Config Realm Environment Variables. More info here: https://github.com/defenseunicorns/uds-identity-config/blob/main/docs/CUSTOMIZE.md#templated-realm-values
 realmInitEnv:
   GOOGLE_IDP_ENABLED: false
-  # Other UDS Identity Config fields that will be used in the realm.json initalization of keycloak
-  # GOOGLE_IDP_CLIENTID: ""
-  # GOOGLE_IDP_CLIENT_SECRET: ""
+#   GOOGLE_IDP_ID: ""
+#   GOOGLE_IDP_SIGNING_CERT: ""
+#   GOOGLE_IDP_NALE_ID_FORMAT: ""
```

Comment:
Is this supposed to be NAME? Just realized when typing up release announcement...

Reply:
https://github.com/defenseunicorns/uds-core/pull/462